api/router.go

```diff
@@ -51,6 +52,12 @@ func Router(c *ctx.AptlyContext) http.Handler {
 		MetricsCollectorRegistrar.Register(router)
 	}
 
+	if c.Config().ServeInAPIMode {
```

Comment:
@mmianl neat!

I have a question, though... `c.Config().ServeInAPIMode` on line 55 that you introduced earlier and `context.Config().FileSystemPublishRoots` on line 56 are, actually from the same context. I'd rather use the same variable for tha sake of consistency. (There are more of such cases down the file, I know, but they are out of scope)

Other than that, LGTM!

Reply:
Fair enough